### PR TITLE
Run lerna bootstrap on postinstall

### PR DESCRIPTION
**Pull request by thejameskyle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
   },
   "private": true,
   "scripts": {
+    "postinstall": "lerna bootstrap",
```
Comment (thejameskyle) [inline package.json]: This way people just have to run `yarn`

File: package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     "iflow-lodash": "^1.1.24",
     "iflow-react-router": "^1.2.1",
     "jest": "^19.0.2",
-    "lerna": "^2.0.0-beta.38",
+    "lerna": "2.0.0-beta.38",
```
Comment (thejameskyle) [inline package.json]: Since we're gonna be using the local version of lerna we should depend on the exact version

**Comment by KyleAMathews** [comment]
Ah cool! Sounds like a good idea.

Seeing this error in Netlify — know what it means?

```bash
5:39:44 PM: Build started
5:40:58 PM: Fetching cached dependencies
5:40:58 PM: Starting to download cache of 204.0MB
5:41:05 PM: Preparing Git Reference pull/735/head
5:41:09 PM: Building site
5:41:09 PM: Running build command
5:41:12 PM: v6.10.0 is already installed.
5:41:13 PM: Using version v6.10.0 of node
5:41:13 PM: Using /opt/buildhome/.rvm/gems/ruby-2.1.2
5:41:14 PM: Installing NPM modules using Yarn version 0.18.1
5:41:14 PM: yarn install v0.18.1
5:41:14 PM: [1/4] Resolving packages...
5:41:15 PM: [2/4] Fetching packages...
5:41:16 PM: warning fsevents@1.0.15: The platform "linux" is incompatible with this module.
5:41:16 PM: info "fsevents@1.0.15" is an optional dependency and failed compatibility check. Excluding it from installation.
5:41:16 PM: [3/4] Linking dependencies...
5:41:20 PM: [4/4] Building fresh packages...
5:41:20 PM: $ lerna bootstrap
5:41:21 PM: Lerna v2.0.0-beta.38
5:41:21 PM: Bootstrapping 24 packages
5:41:21 PM: Preinstalling packages
5:41:21 PM: Installing external dependencies
5:41:23 PM: NpmUtilities.installInDir ("./packages/gatsby-parser-json", ["babel-cli@^6.18.0","bluebird@^3.4.6","unist-util-select@^1.5.0"],...)
Command exited with status 1: yarn install
5:41:23 PM: Errored while running BootstrapCommand.execute
Command exited with status 1: yarn install
5:41:23 PM: Waiting for 3 child processes to exit. CTRL-C to exit immediately.
5:41:23 PM: NpmUtilities.installInDir ("./packages/gatsby-parser-sharp", ["babel-cli@^6.18.0","bluebird@^3.4.6","unist-util-select@^1.5.0"]...)
Command exited with status 1: yarn install
5:41:35 PM: error Command failed with exit code 1.
5:41:36 PM: info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
5:41:36 PM: Error during Yarn install
5:41:36 PM: Cleaning up docker container
5:40:23 PM: Error running command: Build script returned non-zero exit code: 1
5:41:36 PM: Error running command: Build script returned non-zero exit code: 1
5:41:38 PM: Finished processing build request
```

**Comment by thejameskyle** [comment]
Yeah I've seen random errors like that. I think Yarn in Lerna is just broken. They kinda hacked in there despite it being literally impossible to implement correctly.

**Comment by jquense** [comment]
I've run into similar issues using Lerna bootstrap on a postinstall hook. I'm not sure why but it seems like the bootstrap starts before Yarn finishes the install. maybe a difference in how the lifecycle hooks run? ghosts?

**Comment by jquense** [comment]
to be clear I don't see the same issue with the hook on a normal npm install

**Comment by KyleAMathews** [comment]
Ok so if we do this we should force npm for now?

**Comment by KyleAMathews** [comment]
@thejameskyle also could you update CONTRIBUTING.md to account for this change.

**Comment by thejameskyle** [comment]
Done

**Comment by thejameskyle** [comment]
Might want to just delete the `yarn.lock`'s inside of the packages. They were never correct anyways.

**Comment by KyleAMathews** [comment]
Looks like Netlify is happy now :-) I'll delete the yarn.locks and merge this. Thanks!

**Comment by jlengstorf** [comment]
Hiya @jamiebuilds! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**